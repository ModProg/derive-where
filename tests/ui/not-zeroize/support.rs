use std::marker::PhantomData;

use derive_where::derive_where;

#[derive_where(Zeroize)]
struct Unsupported<T>(PhantomData<T>);

fn main() {}
