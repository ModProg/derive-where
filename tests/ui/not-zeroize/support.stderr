error: requires crate feature `zeroize`
 --> tests/ui/not-zeroize/support.rs:5:16
  |
5 | #[derive_where(Zeroize)]
  |                ^^^^^^^
